rest/taskana-rest-spring/src/main/java/pro/taskana/common/rest/QueryHelper.java

```diff
@@ -0,0 +1,80 @@
+package pro.taskana.common.rest;
```

Comment:
This package ok?

Reply:
This package is part of the public API, isn't it? I don't think that the QueryHelper belongs to the public API.